.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: HACF Discord
+    url: https://discord.gg/PaZFEjX
+    about: The HACF discord server
+  - name: HACF Forums
+    url: https://forum.hacf.fr/
+    about: The HACF forum
```

Comment:
To motivate user to use Discord and Forum, can you put details in the `about` key explaining why they should use the Forum. It could be good idea to specify the Discord is french speaking.